certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1365,12 +1365,14 @@ public class Script : ScriptBase
     int recipientCount = envelope["recipients"]["recipientCount"].ToObject<int>();
     var recipientCountInNaturalLanguage = (recipientCount > 1) ?
         (" and " + (recipientCount - 1).ToString() + " others have ") : " "; 
-
+ 
     JArray documentArray = (envelope["envelopeDocuments"] as JArray) ?? new JArray();
-    var documentCountInNaturalLanguage = (documentArray.Count > 1) ?
-      (" and " + (documentArray.Count - 1).ToString() + " other documents ") : " ";
+    var documentCountInNaturalLanguage = (documentArray.Count == 2) ? ""
```

Comment:
Looks hard to read. Consider refactoring.
`var documentCount = documentArray.Count;
string documentCountInNaturalLanguage = "";

if (documentCount == 3)
{
    documentCountInNaturalLanguage = $" and 1 other document";
}
else if (documentCount > 3)
{
    documentCountInNaturalLanguage = $" and {documentCount - 2} other documents";
}
`

Reply:
Fixed it.